single_cell_RNAseq_snake/README.md

```diff
@@ -0,0 +1,72 @@
+# What
+
+This directory is meant to contain single cell pre-processing code using the snakemake workflow management system. 
+
+Snakemake is being evaluated as as alternative to nextflow.
+
+## Snakemake set-up
+
+### How was snakemake installed? 
+
+`mamba` is the recommended way to install snakemake. It is a conda distribution that some optimized features suitable for snakemake. 
+
+Miniforge is a repository that holds the minimal install for Conda.
+
+The conda distribution is installed in `/krummellab/data1/software/miniforge3` and the utilities like `mamba` live in: `/krummellab/data1/software/miniforge3/bin`.
+
+### Usage
+
+Add `/krummellab/data1/software/miniforge3/bin` to your path.
+
+Run `conda activate snakemake` to enter the conda snakemake virtual environment.
+
+Run `snakemake -h` to verify that you can reference snakemake globally.
+
+### What is going on?
```

Comment:
This is a great explanation -- but I think most folks know how to use conda in DSCoLab. These instructions could be moved to additional documentation for how to use conda, and removed from the README.

Reply:
> I think most folks know how to use conda in DSCoLab

This is good to know! It is actually my first time using conda (I mostly use virtual-env + pip3) so this was mostly for me to remember haha. 

Given that we might use this pipeline for our first Vulcan v2 run, and I'm not sure if the other DL engineers are familiar with conda - I'm going to leave it in for now, but happy to move it somewhere else in the future :)

Reply:
Makes sense -- one comment that I'll make is that most folks have their own miniconda repository installed or are using the CBI one (`module load CBI conda`) so this requires each user changing where their conda default comes from. I'm not sure what makes sense to get around this but just something to consider.